src/scss/index.scss

```diff
@@ -103,5 +103,27 @@ $color-selection-bg           : inherit;
 // <hr> border color
 $hr-color                     : $grey--lightest;
 
+// Brand ray colors
+
+$brand-rays--blue             : #2f7de1;
```

Comment:
Super NIT.  Could you put the brand rays colours just below the `// Brand specific colours` block.

Otherwise LGTM